src/string-builder.js

```diff
@@ -52,4 +67,34 @@ StringBuilder.prototype.catIf = function(...args) {
     }
 
     return this;
+};
+
+StringBuilder.prototype.wrap = function(prefix, suffix) {
+    this.decorators.push('wrap');
+    if (prefix) {
+        this.prefix = prefix;
+    }
+
+    if (suffix) {
+        this.suffix = suffix;
+    }
+
+    return this;        
+};
+
+StringBuilder.prototype.end = function(deep) {   
+    if(deep) {
```

Comment:
The deep shall be the `n` number of *effects* or *decorators* you want to remove, e.g.:
- `deep === 1`, remove the last effect name it `wrap`, `suffix` or `prefix`
- `deep === 2`, remove the last two effects from the stack.
- `deep === undefined`, remove the last effect...